blockly-core/core/ui/block.js

```diff
@@ -17,6 +17,8 @@
  * limitations under the License.
  */
 
+/* global Blockly, goog */
```

Comment:
Oh, hmm. That might just be a product of my own development environment; looks like we don't lint blockly-core at all.